modules/calendar/add_edit_event.php

```diff
@@ -57,25 +57,44 @@
  $userid        = $_GET['userid'];
  $default_catid = $_GET['catid'] ? $_GET['catid'] : '5';
  //
- if ($date)
+if ($date) {
   $date = substr($date, 0, 4) . '-' . substr($date, 4, 2) . '-' . substr($date, 6);
- else
+} else {
   $date = date("Y-m-d");
+}
+
  //
  $starttimem = '00';
- if (isset($_GET['starttimem']))
+if (isset($_GET['starttimem'])) {
   $starttimem = substr('00' . $_GET['starttimem'], -2);
+}
+ $endtimem = '00';
+if (isset($_GET['endtimem'])) {
+  $endtimem = substr('00' . $_GET['endtimem'], -2);
+}
  //
  if (isset($_GET['starttimeh'])) {
   $starttimeh = $_GET['starttimeh'];
   if (isset($_GET['startampm'])) {
-   if ($_GET['startampm'] == '2' && $starttimeh < 12)
+        if ($_GET['startampm'] == '2' && $starttimeh < 12) {
```

Comment:
This a 12 hour time format fix?  Do people still use 12 hour time format? 😜

Reply:
Yes